src/routes/dashboard/containers/DashboardContainer.js

```diff
@@ -61,5 +61,5 @@ export default compose([
   branch(props => props.error, renderComponent(Failure)),
   branch(props => props.invalidLanguage, renderComponent(LanguageNotAvailableContainer)),
   translate('dashboard'),
-  withTheme
+  withTheme(props => props.language)
```

Comment:
It's good that we get rid of `withTheme`. Flow types are bad for it... Like for translate actually.